### PR TITLE
Boilerplate

### PR DESCRIPTION
**Pull request by tratnayake**
This the basic skeleton for the front end taken from https://github.com/StephenGrider/ReduxSimpleStarter

Added containers folder and a splash page with icon.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="/style/style.css">
+    <link rel="stylesheet" href="https://cdn.rawgit.com/twbs/bootstrap/48938155eb24b4ccdde09426066869504c6dab3c/dist/css/bootstrap.min.css">
+    <script src="https://maps.googleapis.com/maps/api/js"></script>
```
Comment (AndrewBurian) [inline index.html:7]: Do you need the google maps api?

File: package.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
+{
+  "name": "redux-simple-starter",
```
Comment (AndrewBurian) [inline package.json]: Rename the package

File: package.json [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  "version": "1.0.0",
```
Comment (AndrewBurian) [inline package.json]: not a 1.0.0 release yet

File: package.json [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+  "description": "Simple starter package for Redux with React and Babel support",
```
Comment (AndrewBurian) [inline package.json]: also rename

File: package.json [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+  "main": "index.js",
+  "repository": "git@github.com:StephenGrider/ReduxSimpleStarter.git",
```
Comment (AndrewBurian) [inline package.json]: And repoint the repo

File: package.json [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+  "scripts": {
+    "start": "node ./node_modules/webpack-dev-server/bin/webpack-dev-server.js",
+    "test": "mocha --compilers js:babel-core/register --require ./test/test_helper.js --recursive ./test",
+    "test:watch": "npm run test -- --watch"
+  },
+  "author": "",
+  "license": "ISC",
```
Comment (AndrewBurian) [inline package.json:13]: Do you want to use the ISC license?

File: src/components/splash_index.js [hunk 7]
```diff
@@ -0,0 +1,13 @@
+import React, { Component } from 'react';
+
+export default class SplashPage extends Component {
+  render() {
+    return (
+      <div>
+        <h1>Roomie</h1>
+        <h3>The app for room-mates</h3>
```
Comment (AndrewBurian) [inline src/components/splash_index.js:8]: *roommates
no hyphen

File: README.md [hunk 8]
```diff
@@ -1 +0,0 @@
-# roomie-webapp
```
Comment (AndrewBurian) [inline README.md:1]: why delete the readme?

File: webpack.config.js [hunk 9]
```diff
@@ -0,0 +1,26 @@
+module.exports = {
+  entry: [
+    './src/index.js'
+  ],
+  output: {
+    path: __dirname,
+    publicPath: '/',
```
Comment (AndrewBurian) [inline webpack.config.js]: Should consider namespacing the public path under `/dist` or `/static` so it doesn't pollute the root level and end up conflicting with real routes we want to use later

**Review by AndrewBurian — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (tratnayake) [reply index.html:7]: Nah, you're right. Should take that out.

**Pull request merged**